### ISSUE TITLE
Download and links shows resource format twice

### ISSUE THREAD

**Issue by ianwallen** [metadata101/iso19139.ca.HNAP#335]
If you look at the following, you can see that the resource format "Supporting Document;PDF;fra" twice

![image](https://github.com/metadata101/iso19139.ca.HNAP/assets/1868233/f44e7e86-fa63-48d7-9b93-db2c995c3eed)

This is the description portion of the format.

```
<gmd:description xsi:type="gmd:PT_FreeText_PropertyType">
<gco:CharacterString>Supporting Document;PDF;fra</gco:CharacterString>
<gmd:PT_FreeText>
<gmd:textGroup>
<gmd:LocalisedCharacterString locale="#fra">Document de soutien;PDF;fra</gmd:LocalisedCharacterString>
</gmd:textGroup>
</gmd:PT_FreeText>
</gmd:description>
```

**Comment by jodygarnett** [comment]
@carhenkel this ticket may need your input :)

**Comment by ianwallen** [comment]
May be related to the following PR 
https://github.com/geonetwork/core-geonetwork/pull/6798

**Comment by CarHenkel** [comment]
@jodygarnett @ianwallen Yes.  This is the same for ECCC.  I didn't realize this wasn't the expected result.  It's been this way as long as I've been on this project... So at least Fall 2021
